src/olympia/blocklist/tests/test_commands.py

```diff
@@ -214,17 +214,20 @@ def test_command(self):
             addon_factory()
         # one version, 0 - *
```

Comment:
Should there be tests added for unsigned and non-webextension add-ons? We seem to have tests for the types of add-ons that should be exported, but not for those that should no longer be exported. Or are those covered by `test_mlbf.py`?

Reply:
the tests in `test_mlbf.py` should cover it.  The setup here is just creating enough `Blocks` to generate the filter - it probably doesn't need as many as we have here at the moment.